cogeo_mosaic/backends/base.py

```diff
@@ -116,6 +118,12 @@ def update(
 
         bounds = bbox_union(new_mosaic.bounds, self.mosaic_def.bounds)
 
+        if self.mosaic_def.mosaicjson != new_mosaic.mosaicjson:
+            warnings.warn(
+                f"Updating `mosaicjson` version from {self.mosaic_def.mosaicjson} to {new_mosaic.mosaicjson}"
+            )
+            self.mosaic_def.mosaicjson = new_mosaic.mosaicjson
```

Comment:
I think it's fine to assume that when you `update` a mosaic using this version of cogeo-mosaic, the `mosaicjson` version will be updated